packages/web-components/fast-element/src/components/element-controller.ts

```diff
@@ -33,6 +33,13 @@ export interface ElementControllerStrategy {
     new (element: HTMLElement, definition: FASTElementDefinition): ElementController;
 }
 
+const enum ConnectionState {
```

Comment:
Should we use our alternative enum pattern for this? We've been using a pattern with frozen objects and some type magic. Take a look at `DOMAspect` for an example.

Reply:
If we're exposing externally that makes sense; Internally I think this pattern is better because it compiles away